lib/pluginmanager/templates/filter-plugin/logstash-filter-example.gemspec.erb

```diff
@@ -0,0 +1,23 @@
+Gem::Specification.new do |s|
+  s.name          = 'logstash-filter-<%= plugin_name %>'
+  s.version       = '0.1.0'
+  s.licenses      = ['Apache License (2.0)']
+  s.summary       = %q{TODO: Write a short summary, because Rubygems requires one.}
```

Comment:
`%q` is just another way of doing string definition in ruby, but I can use also regular strings if this is more easy to understand.